tiproxy/tiproxy-configuration.md

```diff
@@ -15,8 +15,9 @@ max-connections = 100
 [api]
 addr = "0.0.0.0:3080"
 
-[log]
-level = "info"
```

Comment:
Is there a specific reason for removing `log.level` from the config example?

Reply:
It's just an example. I don't want to make the example too long.
`log.level` is `info` by default, but typically users need to set `virtual-ip`.